index.Rmd

```diff
@@ -562,7 +562,7 @@ gentri_crime_tract_1=gentri_crime_tract_1[complete.cases(gentri_crime_tract_1[,"
 
 Add any additional processing steps here.
 
-# Results
```

Comment:
I'm curious why in the website comments such as "## PhantomJS not found. You can install it with webshot::install_phantomjs(). If it is installed, please make sure the phantomjs executable can be found via the PATH variable." were included in the final text?

Reply:
In terms of your maps, I think you would benefit from a more explanatory title and adjusting the x axis so it's legible and doesn't overlap. I also think the labels should be adjusted so that you can understand what gendegrees is without having to go and read the code. The leaflet map would benefit from a static legend.